realm/realm-library/src/main/cpp/jni_util/log.hpp

```diff
@@ -184,7 +184,6 @@ class CoreLoggerBridge : public realm::util::RootLogger {
     static CoreLoggerBridge& shared();
```

Comment:
Because of `SyncLoggerFactory`, singleton is not suitable for this class anymore. Just remove `shared()`. Actually we can also separate tags for core and sync. But it is not relevant to this PR. I can do it later.